sys/include/net/sock/dtls.h

```diff
@@ -435,6 +437,7 @@ void sock_dtls_init(void);
  * @param[in] udp_sock  Existing UDP sock to be used underneath
  * @param[in] tag       Credential tag of the sock. The sock will only use
  *                      credentials with the same tag given here.
+ *                      @see net_credman.
```

Comment:
Since this is rendered outside of the parameter doc anyway, I would prefer this to be put before the parameter doc.